hokusai/production.yml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
is this just weird diff? Just want to check in here and make sure the file is still valid.